composeApp/src/desktopTest/kotlin/data/MovieRepositoryImplTest.kt

```diff
@@ -0,0 +1,111 @@
+package data
+
+import edu.dyds.movies.data.MovieRepositoryImpl
+import edu.dyds.movies.data.external.MoviesExternalSource
+import edu.dyds.movies.data.local.MoviesLocalSource
+import edu.dyds.movies.domain.entity.Movie
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+
+class MovieRepositoryImplTest {
+
+    private lateinit var localSource: MoviesLocalSource
+    private lateinit var externalSource: MoviesExternalSource
+    private lateinit var repository: MovieRepositoryImpl
+    private val fakeMovie =
+        Movie(
+            id = 1,
+            title = "Movie 1",
+            overview = "Some overview",
+            releaseDate = "2023-01-01",
+            poster = "poster.jpg",
+            backdrop = "backdrop.jpg",
+            originalTitle = "Original Movie 1",
+            originalLanguage = "en",
+            popularity = 9.8,
+            voteAverage = 8.7
+        )
+
+    @BeforeTest
+    fun setUp() {
+        localSource = mockk()
+        externalSource = mockk()
```

Comment:
Tip: mockk tiene un parámetro para no tener que hacer el "every" de funciones que retornan unit
```suggestion
         localSource = mockk(relaxUnitFun = true)
        externalSource = mockk(relaxUnitFun = true)
```
Así se evitan los every como `coEvery { localSource.clear() } returns Unit`